backup-all.sh

```diff
@@ -30,10 +31,3 @@ source backup-volumes.sh
 
 echo ""
 
-echo - upload to dropbox
```

Comment:
The same with dropbox, if you make it optional and more generic by setting a var or argument, I'll consider it